paasta_tools/utils.py

```diff
@@ -2995,6 +2996,27 @@ def get_hostname() -> str:
     return socket.getfqdn()
 
 
+def get_files_of_type_in_dir(
+    service: str = None, soa_dir: str = DEFAULT_SOA_DIR, file_type: str = None
+) -> List[str]:
+    """Recursively search path if type of file exists.
+
+    :param file_name: a string of a type of a file (kubernetes, slo, etc.)
+    :param path: a string path
+    :return: a list
+    """
+    service = "**" if service is None else service
+    soa_dir = DEFAULT_SOA_DIR if soa_dir is None else soa_dir
+    file_type += "-*.yaml"
+    return [
+        file_path
+        for file_path in glob.glob(
+            os.path.join(soa_dir, service, file_type),
+            recursive=False,
```

Comment:
does this not need to be True for ** to recurse?

Reply:
Ah right. This only captures slos written outside of a slo directory. Changed to True!